src/metabase/api/metric.clj

```diff
@@ -25,6 +26,13 @@
   (check-superuser)
   (check-404 (metric/retrieve-metric id)))
 
+(defendpoint GET "/"
+  "Fetch all `Metrics`. You must be a superuser to do this."
+  [id]
+  (check-superuser)
```

Comment:
Why's this a superuser only? 

The data model reference is in userland, not the admin.